doc/reference/trd1-trds.md

```diff
@@ -247,23 +249,24 @@ from TinyOS Enhancement Proposal 1 (TEP 1).
 7. Author's Address
 ====================================================================
 
-```
-Philip Levis
-409 Gates Hall
-Stanford University 
-Stanford, CA 94305
+    Philip Levis
+    409 Gates Hall
+    Stanford University
+    Stanford, CA 94305
 
-phone - +1 650 725 9046
+    phone - +1 650 725 9046
 
-email - pal@cs.stanford.edu
-```
+    email - pal@cs.stanford.edu
 
 
 8. Citations
 ====================================================================
 
-[TRD1]: <trd1-trds.md>
-[Markdown]: <https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links>
+<a name="TRD1" />[TRD1]
```

Comment:
Yeah, that's what I was thinking too (it's a lot easier to type the markdown than the html, for one) but I talked to @phil-levis and he seemed to prefer something like this.  I forget exactly why; let's see what he says.